src/vue/forms/InvestForm.vue

```diff
@@ -158,23 +158,6 @@
       </form>
     </template>
 
-    <template v-else-if="isLoaded && isAllowedAccountType">
-      <no-data-message
-        icon-name="alert-circle"
-        :title="'invest-form.insufficient-balance-title' | globalize"
-        :message="'invest-form.insufficient-balance-desc' | globalize"
-      >
-        <router-link
-          :to="vueRoutes.movements"
-          tag="button"
-          type="button"
-          class="app__button-raised invest-form__discover-tokens-btn"
```

Comment:
Now we have useless `invest-form__discover-tokens-btn` in the CSS. When you deleted some stuff, make sure that you deleted it from everywhere